src/main/java/com/incognia/onboarding/RegisterSignupRequest.java

```diff
@@ -5,12 +5,13 @@
 import java.util.List;
 import lombok.Builder;
 import lombok.Value;
+import org.jetbrains.annotations.Nullable;
```

Comment:
duvida: O import do nullable é esse mesmo?

Reply:
In this case it is, we do'nt use jakarta/javax here